sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -32,6 +32,7 @@
     private static final String SUPPRESSED_PROPERTY_KEY = "suppressed";
     private static final String UNINSTALLING_PROPERTY_KEY = "uninstalling";
     private static final String LAST_COMPLETED_UPDATE_TYPE_KEY = "last-completed-update-type";
+    private static final String PROPERTY_TASK_INFO_SUFFIX = ":taskStatus";
```

Comment:
@gabrielhartmann Didn't you ask for this?

I'm fine with brining it into alignment with properties.